ios/iosApp/UI/BaseViewModelView.swift

```diff
@@ -0,0 +1,15 @@
+import TrikotFrameworkName
+import UIKit
+
+class BaseViewModelView<VM: ViewModel>: UIView {
```

Comment:
Again, would drop the `Base` prefix and let the app subclass it to include a "base" definition according to their needs